django-stubs/contrib/admin/decorators.pyi

```diff
@@ -1,6 +1,12 @@
-from typing import Any, Callable, Optional, Type
+from typing import Callable, Optional, Type, Sequence
 
 from django.contrib.admin.sites import AdminSite
 from django.db.models.base import Model
 
+def action(
+    function: Optional[Callable] = None,
```

Comment:
Can we describe the internals of `Callable` here? Maybe partially: just params or return type? Or it can be literally any callable?

Reply:
Btw, we use `...` instead of plain values.

Reply:
> Can we describe the internals of Callable here? Maybe partially: just params or return type? Or it can be literally any callable?

Sure!

I found some other stuff from `contrib.admin` that was missing as well.